Description: Don't embed parts of jmapviewer in final josm JAR.
Author: David Paleino <dapal@debian.org>
Bug-Debian: https://bugs.debian.org/731029
Forwarded: not-needed

--- a/build.xml
+++ b/build.xml
@@ -211,7 +211,6 @@ Debian-Release: ${debian.version}
             <zipfileset dir="images" prefix="images"/>
             <zipfileset dir="data" prefix="data"/>
             <zipfileset dir="styles" prefix="styles"/>
-            <zipfileset dir="${src.dir}/org/openstreetmap/gui/jmapviewer/images" prefix="org/openstreetmap/gui/jmapviewer/images"/>
         </jar>
     </target>
     <!-- Mac OS X target -->
